tests/e2e/endpoint-extraction/00-assert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
```

Comment:
I see there is a unit test for the same `func` that you are modifying the logic i.e. `TestReceiverParsePortFromEndpoint` in `receiver_test.go` I would recommend adding an additional case in the test function rather than creating an e2e suite for a corner/edge case.

Reply:
@yuriolisa I don't think you need these tests as you added them into unit tests.